README.md

```diff
@@ -25,6 +36,122 @@ sudo php -d set_time_limit=3600 -d memory_limit=1024M bin/magento setup:di:compi
 ## Configure
 In the admin panel under store configuration, you can set the  options for the extension (Stores->Configuration->Tealium->Tags). You will need to enable it, and define your TiQ account, profile, and environment information.
 
+## How To Use
+### Predefined data layer variables already available
+#### Home
+- site_region
+- site_currency
+- page_name
+- page_type
+#### Search
+- site_region
+- site_currency
+- page_name
+- page_type
+- search_results
+- search_keyword
+#### Category
+- site_region
+- site_currency
+- page_name
+- page_type
+- page_section_name
+- page_category_name
+- page_subcategory_name
+#### ProductPage
+- site_region
+- site_currency
+- page_name
+- page_type
+- product_id
+- product_sku
+- product_name
+- product_brand
+- product_category
+- product_unit_price
+- product_list_price
+#### Cart
+- site_region
+- site_currency
+- page_name
+- page_type
+- product_id
+- product_sku
+- product_name
+- product_quantity
+- product_list_price
+- Confirmation
+- site_region
+- site_currency
+- page_name
+- page_type
+- order_id
+- order_discount
+- order_subtotal
+- order_shipping
+- order_tax
+- order_payment_type
+- order_total
+- order_currency
+- customer_id
+- customer_email
+- product_id
+- product_sku
+- product_name
+- product_list_price
+- product_quantity
+- product_discount
+- product_discounts
+- site_region
+- site_currency
+- page_name
+- page_type
+- customer_id
+- customer_email
+- customer_type
+### Adding Custom Data Sources
```

Comment:
This language refers to legacy features in Tealium IQ. OK to include now but we should consult our documentation team for their collaboration in the future.